ui/raidboss/data/04-sb/ultimate/unending_coil_ultimate.ts

```diff
@@ -1485,6 +1646,57 @@ const triggerSet: TriggerSet<Data> = {
         },
       },
     },
+    {
+      id: 'UCU Heavensfall Nael Spot',
+      type: 'StartsUsing',
+      // Grab position data once Bahamut begins casting Megaflare Dive
+      netRegex: { id: '26E1', source: 'Bahamut Prime', capture: false },
+      condition: (data) => data.trio === 'heavensfall',
+      promise: async (data) => {
+        if (!data.trioSourceIds.nael || !data.trioSourceIds.twin || !data.trioSourceIds.bahamut)
+          return;
+        data.combatantData = [];
+        data.combatantData = (await callOverlayHandler({
+          call: 'getCombatants',
+          ids: [data.trioSourceIds.nael, data.trioSourceIds.bahamut, data.trioSourceIds.twin],
+        })).combatants;
+      },
+      alertText: (data, _matches, output) => {
+        // Bosses line up adjacent to one another, but don't necessarily have discrete directional positions (based on 8Dir scale).
+        // But we can calculate their position as an angle (relative to circular arena): 0 = N, 90 = E, 180 = S, 270 = W, etc.
+        // Set initial angles to -1 to avoid issues with nullish checks, as 0 is a valid angle and a nullish value.
+        let naelAngle = -1;
+        let bahamutAngle = -1;
+        let twinAngle = -1;
+        let naelPos = 'unknown';
+        for (const mob of data.combatantData) {
+          const mobAngle = (Math.round(180 - 180 * Math.atan2(mob.PosX, mob.PosY) / Math.PI) % 360);
+          // As OP does not return combatants in the order, they were passed, match based on sourceId.
+          if (mob.ID === data.trioSourceIds.nael)
+            naelAngle = mobAngle;
+          else if (mob.ID === data.trioSourceIds.bahamut)
+            bahamutAngle = mobAngle;
+          else if (mob.ID === data.trioSourceIds.twin)
+            twinAngle = mobAngle;
+        }
+        if (naelAngle >= 0 && bahamutAngle >= 0 && twinAngle >= 0) {
+          if (isClockwise(naelAngle, bahamutAngle))
+            naelPos = isClockwise(naelAngle, twinAngle) ? 'left' : 'middle';
+          else
+            naelPos = isClockwise(naelAngle, twinAngle) ? 'middle' : 'right';
+        }
+        return output.naelPosition!({ dir: output[naelPos]!() });
+      },
+      outputStrings: {
+        naelPosition: {
+          en: 'Nael is ${dir}',
```

Comment:
```suggestion
          en: '${dir} Nael',
```

Just rewording to put the important word first.